ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/configs/CassandraConfigurationBase.java

```diff
@@ -75,4 +75,10 @@
 
     @DefaultValue("true")
     Boolean getCreateSchema();
+
+    @PropertyName(name = "truststorePath")
+    String getTruststorePath();
+
+    @PropertyName(name = "truststorePass")
```

Comment:
@PropertyName annotations here are redundant since the overridden name is not different from the method name